tests/test_enterprise/api/test_views.py

```diff
@@ -1760,11 +1764,11 @@ def test_enterprise_customer_course_enrollments_detail_success(
                     payload.get('course_mode'),
                     cohort=payload.get('cohort'),
                 )
-        elif 'user_email' in post_data:
+        elif 'user_email' in payload and payload.get('is_active', True):
```

Comment:
Note: The test code in this branch was not executing.  The changes here are my additions and changes needed to get this branch working in the test.